src/site_assessment/geom_ops.jl

```diff
@@ -165,9 +164,8 @@ function find_horizontal(geom::GIWrap.Polygon)::Vector{Tuple{Float64,Float64}}
     coords = collect(GI.coordinates(geom)...)
     first_coord = first(coords)
     second_coord = coords[
-        (getindex.(coords, 2).∈first_coord[2]) .&&
-        (getindex.(coords, 1).∉first_coord[1])
-    ]
+    (getindex.(coords, 2) .∈ first_coord[2]) .&& (getindex.(coords, 1) .∉ first_coord[1])
+]
```

Comment:
I like the old code better :(
seems like a JuliaFormatter bug that it removes indent on closing bracket.

Is there a better way to write this code? or I could disable formatting here

Reply:
Created issue https://github.com/domluna/JuliaFormatter.jl/issues/876